components/scream/src/diagnostics/register_diagnostics.hpp

```diff
@@ -43,6 +44,7 @@ inline void register_diagnostics () {
   diag_factory.register_product("ShortwaveCloudForcing",&create_atmosphere_diagnostic<ShortwaveCloudForcingDiagnostic>);
   diag_factory.register_product("LongwaveCloudForcing",&create_atmosphere_diagnostic<LongwaveCloudForcingDiagnostic>);
   diag_factory.register_product("RelativeHumidity",&create_atmosphere_diagnostic<RelativeHumidityDiagnostic>);
+  diag_factory.register_product("ZonalVapFlux",&create_atmosphere_diagnostic<RelativeHumidityDiagnostic>);
```

Comment:
You need to change `RelativeHumidyDiagnostic` after the `create_atmosphere_diagnostic` call.